Vagrantfile

```diff
@@ -133,7 +141,6 @@ Vagrant.configure("2") do |global_config|
       end
 
       config.vm.provision :chef_solo do |chef|
-        chef.product = "chef-workstation"
```

Comment:
Is this redundent? ie. can we leave this off all other flavours? ie Virtualbox etc? I guess I'll have to give it a spin. Otherwise it would be better if we added this conditionally. We don't want to ONLY support parrellels :P

Reply:
OK without this like it failed to build my VirtualBox jammy. So this is required for other flavours/provisioners. So it need to be make conditional. Maybe something like:

```
config.vm.provision :chef_solo do |chef|
  unless vagrant_box.include? "m1" then
    chef.product = "chef-workstation"
  end
  ...
```